library/public/loader.h

```diff
@@ -12,76 +13,64 @@ namespace f3d
  * @class   loader
  * @brief   Class to load files in F3D
  *
- * A class to load files in F3D.
- * Multiple files and folders can be added using addFiles/addFile and stored
- * in an internal list, with a index pointing to a "current" file.
- * Then files from the list can be loaded one by one using loadFile method.
- * Information about the files in the list can be recovered using getFileInfo.
- * The current file can be changed either on loadFile with a dedicated enum or
- * by using setCurrentFileIndex.
+ * A class to load files in F3D. The loader can load a full scene or multiple geometries into a
+ * default scene. It also support checking if a scene or geometry reader is available for a given
+ * file.
+ *
+ * Example usage:
+ *
+ *  std::string path = ...
+ *  f3d::engine eng(f3d::window::Type::NATIVE);
+ *  f3d::loader& load = eng.getLoader();
+ *
+ *  if (load.hasSceneReader(path)
+ *  {
+ *    load.loadFullScene(path);
+ *  }
+ *  else if (load.hasFullSceneReader(path)
+ *  {
+ *    load.loadGeometry(path);
+ *  }
```

Comment:
```suggestion
 *  if (load.hasFullSceneReader(path)
 *  {
 *    load.loadFullScene(path);
 *  }
 *  else if (load.hasSceneReader(path)
 *  {
 *    load.loadGeometry(path);
 *  }
```

Reply:
both version, mine and yours, are both incorrect. I will fix it.